src/core_plugins/input_control_vis/public/lineage/parent_candidates.js

```diff
@@ -0,0 +1,21 @@
+import { getTitle } from '../editor_utils';
+
+export function getParentCandidates(controlParamsList, controlId, lineageMap) {
+  return controlParamsList.filter((controlParams) => {
+    // Ignore controls that do not have index pattern and field set
+    if (!controlParams.indexPattern || !controlParams.fieldName) {
```

Comment:
Is this ignoring controlParams that haven't been "completed"? We have this same check elsewhere, can we generalize it?